site/content/entry/yuzu-progress-report-aug-2020/index.md

```diff
@@ -0,0 +1,209 @@
+
+What this means is that users with CPUs from the AMD Phenom II series or older, Intel 3rd generation Core-i series or older, and all Pentium, Celeron and Atom CPUs can now select the "Unsafe" setting and see major performance improvements in almost all games. Don't bother to try this setting on CPUs that have FMA as it will not improve your performance in that case.
+
+Due to being a less precise alternative, expect Dragons.
+
+{{< imgs
+    "./fma.png| You can find it here"
+  >}}
+
+## UI Changes
+
+As part of the input rewrite, [Rei](https://github.com/Its-Rei) fixed several consistency bugs affecting the `Midnight Blue` UI theme. You can see the results bellow.
+
+{{< single-title-imgs
+    "Input window fixes, before (left) and after (right)"
+    "./input_before.png"
+    "./input_after.png"
+  >}}
+  
+&nbsp;
+  
+{{< single-title-imgs
+    "Gamelist fixes, before (left) and after (right)"
+    "./main_before.png"
+    "./main_after.png"
+  >}}
+
+Both of our dark themes (Dark and Midnight Blue) had a transparency bug affecting the per-game settings pop-up window. [toastUnlimited](https://github.com/lat9nq) made the necessary changes to fix this by [setting QLabel background color to transparent](https://github.com/yuzu-emu/yuzu/pull/4420). You can see the changes below.
+
+{{< single-title-imgs
+    "Dark theme, before (left) and after (right)"
+    "./darkbug.png"
+    "./darkfix.png"
+  >}}
+
+&nbsp;
+
+{{< single-title-imgs
+    "Midnight Blue theme, before (left) and after (right)"
+    "./midnightbug.png"
+    "./midnightfix.png"
+  >}}
+  
+[toastUnlimited](https://github.com/lat9nq) also added an option to [access game specific settings while playing.](https://github.com/yuzu-emu/yuzu/pull/4515) This makes it easy to change graphics accuracy or other settings on the fly.
+
+{{< imgs
+    "./pergame.png| Found it!"
+  >}}
+
+`Force 30FPS` was an option that was used for debugging, and as an emergency brake for the old days when yuzu lacked a proper frame limiter. The fate of this tick-box was undecided for long, moving it to the `Debug` tab was considered at one point. But in the end, it was finally [removed by](https://github.com/yuzu-emu/yuzu/pull/4446) [Morph](https://github.com/Morph1984). This option was no longer functional after the changes of `Project Prometheus` took effect, so this was a necessary sacrifice.
+
+Our users reported that some checkboxes were duplicated after the addition of Vulkan. These were `Dump Decompressed NRO` and `Dump ExeFS` which could be found both in `General > Debug` and `System > Filesystem > Patch Manager`. [Tobi](https://github.com/FearlessTobi) put an end to this clone war, [removing the duplicates in Debug](https://github.com/yuzu-emu/yuzu/pull/4429).
+  
+## Future Projects
+
+[epicboy](https://github.com/ameerj) is working on an universal, GPGPU accelerated method to decode ASTC textures. We shall hear more of this after the Texture Cache rewrite is finished.
```

Comment:
We may want to add `(General-Purpose GPU)` after GPGPU, as it's not a common acronym.